src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/TextEditorContextMenu.cs

```diff
@@ -433,13 +433,6 @@ private class EditorContextMenu : ContextMenu
             // Creates a new instance.
             internal void AddMenuItems(TextEditor textEditor, bool userInitiated)
```

Comment:
can the `userInitiated` parameter be removed from this method and `AddClipboardItems`?